### PR TITLE
Refactor resource policy 

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/ibm_pi_constants.go [hunk 1]
```diff
@@ -29,7 +29,13 @@ const (
 	Arg_VolumeGroupID                       = "pi_volume_group_id"
 	Arg_VolumeID                            = "pi_volume_id"
 	Arg_VolumeOnboardingID                  = "pi_volume_onboarding_id"
-
+	Arg_VPNPolicyDhGroup                    = "pi_policy_dh_group"
```
Comment (ismirlia) [inline ibm/service/power/ibm_pi_constants.go:32]: I would prefer if the name of the variable reflected the value of it.

So "pi_policy_dh_group" would become Arg_PolicyDhGroup.

Comment (Diptipowervs) [reply ibm/service/power/ibm_pi_constants.go:32]: Change made

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPIIKEPolicyBasic (122.15s)
PASS

File: ibm/service/power/ibm_pi_constants.go [hunk 2]
```diff
@@ -29,7 +29,13 @@ const (
...
 	Arg_VolumeID                            = "pi_volume_id"
 	Arg_VolumeOnboardingID                  = "pi_volume_onboarding_id"
-
+	Arg_VPNPolicyDhGroup                    = "Arg_PolicyDhGroup"
+	Arg_VPNPolicyAuthentication             = "pi_policy_authentication"
+	Arg_VPNPolicyEncryption                 = "pi_policy_encryption"
+	Arg_VPNPolicyKeyLifetime                = "pi_policy_key_lifetime"
+	Arg_VPNPolicyName                       = "pi_policy_name"
+	Arg_VPNPolicyPresharedKey               = "pi_policy_preshared_key"
+	Arg_VPNPolicyVersion                    = "pi_policy_version"
```
Comment (michaelkad) [inline ibm/service/power/ibm_pi_constants.go:38]: No need for vpn in the name.
```suggestion
	Arg_PolicyDhGroup                    = "Arg_PolicyDhGroup"
	Arg_PolicyAuthentication             = "pi_policy_authentication"
	Arg_PolicyEncryption                 = "pi_policy_encryption"
	Arg_PolicyKeyLifetime                = "pi_policy_key_lifetime"
	Arg_PolicyName                       = "pi_policy_name"
	Arg_PolicyPresharedKey               = "pi_policy_preshared_key"
	Arg_PolicyVersion                    = "pi_policy_version"
```

**Review by michaelkad — CHANGES_REQUESTED** [review]

**Comment by michaelkad** [comment]
Deprecated

**Pull request closed without merging**